vtk/build.sh

```diff
@@ -32,6 +32,9 @@ if [ `uname` == Darwin ]; then
     CC=cc
     CXX=c++
     PY_LIB="libpython${PY_VER}.dylib"
+    if [ `xcode-select -p` = "/Library/Developer/CommandLineTools" ]; then
+      unset MACOSX_DEPLOYMENT_TARGET
+    fi
```

Comment:
It's not conda which assumes this, it's cmake. I had to dig a lot to find the 10.6 SDK (so I can post it somewhere if people needs it) but we need it to ensure that VTK runs from 10.6 onwards, not from the system version VTK was built in.

So, I don't accept this change. Of course, you can just fork our recipe and add this line if you want, but I'd like to ask you to remove it here.